tmt/base.py

```diff
@@ -778,17 +778,23 @@ def go(self):
         all_steps = self.opt('all') or not enabled_steps
         since = self.opt('since')
         until = self.opt('until')
+        after = self.opt('after')
```

Comment:
@psss should not since and after be mutually exclusive? :)

Reply:
Yes, but I think this is just a cosmetic issue. We can polish later.